site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -259,6 +259,27 @@ and *one-way* commands to the topic:
 
 (For an example of the incoming command see above at authenticated device)
 
+### Receiving Commands (authenticated Gateway)
+
+*Gateway* components can receive commands for devices which do not connect to a protocol adapter directly but instead are connected to the gateway, e.g. using some low-bandwidth radio based technology like [SigFox](https://www.sigfox.com) or [LoRa](https://www.lora-alliance.org/). Corresponding devices have to be configured so that they can be used with a gateway. See [Configuring Gateway Devices]({{< relref "/admin-guide/device-registry-config.md#configuring-gateway-devices" >}}) for details.
+
+If a device is configured in such a way that there can be *one* gateway, acting on behalf of the device, a command sent to this device will by default be directed to that gateway.
+
+If a device is configured to be used with *multiple* gateways, the particular gateway that last acted on behalf of the device will be the target that commands for that device will be routed to. The mapping of device and gateway last used by the device is updated whenever a device sends a telemetry, event or command response message via the gateway. This means that for a device configured to be used via multiple gateways to receive commands, the device first has to send at least one telemetry or event message to establish which gateway to use for receiving commands for that device.
+
+An authenticated gateway MUST use the following topic filter to subscribe to commands:
+
+* `command/+/+/req/#`
+
+**Example**
+
+    mosquitto_sub -v -u 'gw@DEFAULT_TENANT' -P gw-secret -t command/+/+/req/#
+
+The adapter will then publish commands for devices, that the gateway has acted on behalf of, to topic:
+
+* for *Request/Response* commands: `command//${device-id}/req/${req-id}/${command}[/*][/property-bag]`
+* for *one-way* commands: `command//${device-id}/req//${command}[/*][/property-bag]`
```

Comment:
the topic we send commands to cannot contain a property bag, can it?

Reply:
Right, I've removed the `[/*][/property-bag]` part also in the other places on the page.